src/content/learn/sharing-state-between-components.md

```diff
@@ -172,21 +179,23 @@ h3, p { margin: 5px 0px; }
 
 </Sandpack>
 
-Try editing the hardcoded `isActive` values in the `Accordion` component and see the result on the screen.
+Coba edit nilai `isActive` yang telah ditentukan sebelumnya di komponen `Accordion` dan lihat hasilnya di layar.
```

Comment:
```suggestion
Coba ubah nilai `isActive` yang dituliskan langsung di dalam kode komponen `Accordion` dan lihat hasilnya di layar.
```

Let's preserve the "hardcoded" meaning in the translated content because it's part of the vital aspect of this step, and we will eventually replace the hardcoded data with a state in the next step.